pytorch_lightning/core/hooks.py

```diff
@@ -372,6 +372,18 @@ def configure_sharded_model(self) -> None:
 class DataHooks:
     """Hooks to be used for data related stuff."""
 
+    __jit_unused_properties__ = ["prepare_data_per_node"]
```

Comment:
Is this necessary now that it's not a property?

Reply:
good catch! Thank you~